packages/lodestar/src/logger/winston.ts

```diff
@@ -47,7 +47,7 @@ export class WinstonLogger implements ILogger {
     });
     //@ts-ignore
     this._level = LogLevel[options.level];
-    this._silent = false;
+    this._silent = process.env.LODESTAR_SILENCE === 'true';
```

Comment:
Can you add a documentation note somewhere, maybe the README or elsewhere, about the `LODESTAR_SILENCE` env var?

Another thought, can you do a check for if `process` exists? I think in the browser `process` will be `undefined`

Reply:
Good eye on `process` possibly being undefined. I'll update this logic and make note of the envar in the README.